workflow-templates/im-build-dotnet-ci.yml

```diff
@@ -19,6 +15,7 @@ on:
     types: [opened, reopened, synchronize, closed]
 
 env:
+  READ_PKG_TOKEN: ${{ secrets.READ_PKG_TOKEN }}
```

Comment:
Do we want to add a comment saying what level of secret (repo, org, etc.) this is, like we do for other secrets?

Reply:
yes. :(